src/pages/List/Aside/ImgSlider.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+
+class ImgSlider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: [],
+      viewImage: 0,
+    };
+  }
+
+  componentDidMount() {
+    const titleData = this.props.fetchData;
+    const imageData = Object.values(titleData.description_images);
+    this.setState({
+      images: imageData,
+    });
+  }
+  handleSlider = n => {
+    const { images, viewImage } = this.state;
+    let currentNum = viewImage + n;
+    if (currentNum === -1) {
+      currentNum = images.length - 1;
+    } else if (currentNum > images.length - 1) {
+      currentNum = 0;
+    }
+    this.setState({
+      viewImage: currentNum,
+    });
+  };
+  render() {
+    const { images, viewImage } = this.state;
+
+    return (
+      <StWraper>
+        <StButton onClick={() => this.handleSlider(-1)}>
+          <img
+            className="leftArrow"
+            src={'../images/left.png'}
+            alt={'left arrow'}
```

Comment:
jsx를 사용하는 부분이 아니기 때문에, 중괄호로 감싸주시지 않아도 됩니다!
그리고 경로도 다음과 같이 접근할 수 있겠네요!

```suggestion
            src="/images/left.png"
            alt="left arrow"
```

Reply:
넵 수정하였습니다 :)